Lib/lib2to3/fixes/fix_urllib.py

```diff
@@ -13,7 +13,7 @@
                 ("urllib.request",
                     ["URLopener", "FancyURLopener", "urlretrieve",
                      "_urlopener", "urlopen", "urlcleanup",
-                     "pathname2url", "url2pathname"]),
+                     "pathname2url", "url2pathname", "getproxies"]),
```

Comment:
Please add a test for this in `Lib/lib2to3/tests/test_fixers.py`. You can use the existing tests for reference like `self.check(b, a)` where `b` is expected Python 3 transformed code and `a` is the Python 2 code

Reply:
@tirkarthi Done!, I am not sure if that is the correct place for the test, please verify